flash/core/utilities/imports.py

```diff
@@ -99,6 +100,25 @@ def _compare_version(package: str, op, version) -> bool:
 _ICEDATA_AVAILABLE = _module_available("icedata")
 _TORCH_ORT_AVAILABLE = _module_available("torch_ort")
 
+if _PIL_AVAILABLE:
+    from PIL import Image
+else:
+
+    class MetaImage(type):
+        def __init__(cls, name, bases, dct):
+            super().__init__(name, bases, dct)
+
+            cls._Image = None
+
+        @property
+        def Image(cls):
+            warn("Mock object called due to missing PIL library. Please use \"pip install 'lightning-flash[image]'\".")
```

Comment:
Let's raise a `ModuleNotFoundError` here rather than warn. Safe to assume whatever code winds up using it won't work correctly.

Reply:
can't do Module not found error, the property needs to return a None value, since Image.Image is being checked for its type in some part of the code, can't throw an error there.